shardingsphere-agent/shardingsphere-agent-core/src/main/java/org/apache/shardingsphere/agent/core/bytebuddy/transformer/ShardingSphereTransformer.java

```diff
@@ -178,4 +169,22 @@
         }
         return result;
     }
+    
+    private ShardingSphereTransformationPoint<?> getInstanceMethodTransformationPoint(final MethodDescription.InDefinedShape md, final List<InstanceMethodPoint> instanceMethodAroundPoints) {
```

Comment:
Please do not use inline import, just use `InDefinedShape`.
By the way, `md` is not match to `InDefinedShape`, is it?

Reply:
`IndefinedShape` extends the `MethodDescription`. It's ok for naming as `md` .